spec/lib/debt_management_center/shared_examples/debt_service_examples.rb

```diff
@@ -21,6 +21,10 @@
       it 'returns a bad request error' do
         VCR.use_cassette('bgs/people_service/no_person_data') do
           VCR.use_cassette('debts/get_letters_empty_ssn', VCR::MATCH_EVERYTHING) do
+            if flipper_enabled
```

Comment:
**Comment:** I actually disagree with the linter here, having the `expect` test statement on its own line keeps the tests readable and clear.